### ISSUE TITLE
Deserialize Exception thrown for some string

### ISSUE THREAD

**Issue by zerg000000** [juji-io/datalevin#12]
deps.edn
```clojure
{:deps {datalevin {:mvn/version "0.2.13"}}}
```

code for reproduce
```clojure
(require '[datalevin.core :as d])
(def conn (d/create-conn {:id {:db/valueType :db.type/long}
                          :text {:db/valueType :db.type/string}} 
                         "tmp"))
(d/transact! conn [{:text "[7/3, 15:36]"
                    :id 3}])
(d/q '[:find (pull ?e [*])
       :where 
       [?e :id 3]]
     @conn)
```

exception
```
Error printing return value (ExceptionInfo) at datalevin.lmdb.LMDB/get_value (lmdb.clj:680).
Fail to get-value: "Thaw failed against type-id: 78"
```

**Comment by zerg000000** [comment]
seems solved in `0.2.16`. but the corrupted db cannot be recovered.

**Comment by huahaiy** [comment]
Cannot reproduce this.
